readme.md

```diff
@@ -7,7 +7,7 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
 **Contributors:** [batmoo](https://profiles.wordpress.org/batmoo), [joen](https://profiles.wordpress.org/joen), [automattic](https://profiles.wordpress.org/automattic), [potatomaster](https://profiles.wordpress.org/potatomaster), [albertomedina](https://profiles.wordpress.org/albertomedina), [google](https://profiles.wordpress.org/google), [xwp](https://profiles.wordpress.org/xwp), [westonruter](https://profiles.wordpress.org/westonruter)  
 **Tags:** [amp](https://wordpress.org/plugins/tags/amp), [mobile](https://wordpress.org/plugins/tags/mobile)  
 **Requires at least:** 4.7  
-**Tested up to:** 4.9  
+**Tested up to:** 4.9.5  
```

Comment:
This can remain 4.9 actually.:

> Just keep it to the major releases. A plugin that is tested on 4.7 will show as tested up to 4.7.2 as well, after all.

https://make.wordpress.org/plugins/2017/04/05/reminder-how-svn-wordpress-org-works/

Reply:
Oh, thanks. That's interesting.